tests/yam/agama/agama_integration.pm

```diff
@@ -20,15 +20,15 @@ sub run {
     my $self = shift;
     my $test = get_required_var('AGAMA_TEST');
 
-    assert_script_run("playwright test --trace on --project chromium --config /usr/share/e2e-agama-playwright tests/" . $test . ".spec.ts", timeout => 1200);
+    assert_script_run("agama-integration-tests /usr/share/agama/integration-tests/tests/" . $test . ".js", timeout => 1200);
+    save_screenshot();
```

Comment:
here, we could use our POM strategy for reboot with needles in the same way that we have for boot_agama.pm because that save_screenshot should belong to some method, or the one really rebooting or some error handling, but I guess you are in the middle here.

Reply:
> here, we could use our POM strategy for reboot with needles in the same way that we have for boot_agama.pm because that save_screenshot should belong to some method, or the one really rebooting or some error handling, but I guess you are in the middle here.

agreed follow-up on this in https://progress.opensuse.org/issues/165980